packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -62,13 +58,22 @@ class FileKeystoreWallet implements Wallet {
   }
 
   public loadKeystore = (): Keystore => {
-    // TODO: handle fs error
-    const data = this.keyStore.service.readFileSync(MODULE_NAME, `${this.id}.json`)
+    const data = new FileService().readFileSync(MODULE_NAME, this.keystoreFileName())
     return JSON.parse(data) as Keystore
   }
 
-  clear = () => {
-    this.keyStore.clear()
+  saveKeystore = (keystore: Keystore) => {
+    const keystoreToSave = keystore
+    keystoreToSave.id = this.id
+    new FileService().writeFileSync(MODULE_NAME, this.keystoreFileName(), JSON.stringify(keystoreToSave))
```

Comment:
`fileService` could be imported from `src/startup/fileService`

Reply:
What's the difference? `startup/fileService` is a singleton?